README.md

```diff
@@ -37,5 +37,20 @@ If we want to expand the API for connections, we can always add more symbols
 for parsing. One I was thinking about is `\->`, which would denote an
 "ending" connection (to signal destroying the path to reach the current action)
 
+# Potion
 
+1) Know the attributes of your potion
 
+2) Go to **builtin.py** -> **Potion** and find **Potion** class. This is where you initialize the potion. **NOTE:** You don't assume the player is going to use an object. This is merely initialization. Tip: Initialize the logic of your object here. 
+
+3) Do not forget to go to **crpg.py** -> **generate()** iff you are:
+    - creating a new potion type. 
+        - go to the first for loop and look for the first if statement. 
+        - remember to include your new object type in n_types
```

Comment:
I still think this documentation is a bit out of place here.

Reply:
Deleted.